plugins/GSdx/GSTexture11.cpp

```diff
@@ -50,7 +50,7 @@ bool GSTexture11::Update(const GSVector4i& r, const void* data, int pitch)
 {
 	if(m_dev && m_texture)
 	{
-		D3D11_BOX box = {r.left, r.top, 0, r.right, r.bottom, 1};
+		D3D11_BOX box = { (UINT)r.left, (UINT)r.top, 0, (UINT)r.right, (UINT)r.bottom, 1 };
 
```

Comment:
I think ``D3D11_BOX`` requires all members to be ``unsigned int`` , it'd nice to explicitly cast 0 -> 0U & 1-> 1U in order to highlight it.